README.md

```diff
@@ -51,55 +51,63 @@ If you wish to attempt to build a basic visual novel with the existing C++ API,
 
 ### Build instructions
 
-We provide build scripts in the root of the repository which may work for you.
-However, you will need to ensure the dependencies are correctly installed.
+These instructions are based on the Conan package manager. you can download Conan from [here.](https://conan.io/)
 
 #### Linux
-First, you must install the dependencies. Using Ubuntu 20.04, this looks something like this:
+First, you must install the dependencies. Using Ubuntu 20.04 there are extra requirements on top of Conan. With those dependencies, it looks something like this:
 ```
-sudo apt install clang cmake doxygen graphviz g++ libfreetype-dev libglfw3-dev \
-    libglm-dev libgmock-dev libgtest-dev liblua5.3-dev libopenal-dev libsndfile1-dev \
-    libspdlog-dev python3-glad
+sudo apt install clang  libgl-dev xorg-dev libx11-xcb-dev libxcb-render0-dev libxcb-render-util0-dev libxcb-xkb-dev \
+libxcb-icccm4-dev libxcb-image0-dev libxcb-keysyms1-dev libxcb-randr0-dev libxcb-shape0-dev libxcb-sync-dev \
+libxcb-xfixes0-dev libxcb-xinerama0-dev xkb-data libxcb-dri3-dev libxcb-util-dev python3-pip \
 ```
-Then, clone and build NovelRT:
+
+Then, install Conan and our configurations should you require them:
+```
+pip3 install conan
+conan config install https://github.com/novelrt/ConanConfig.git
+```
+
+If you are building from a command line terminal, clone NovelRT and set up the build folder like so:
 ```
 git clone https://github.com/NovelRT/NovelRT.git
 mkdir -p NovelRT/build
 cd NovelRT/build
+conan install .. --build=missing --build=bison --profile linux-clang10-amd64
+```
```

Comment:
Add the note: "all our supported scenarios have been defined via our conan profiles" as per Discord :)